packages/vector_graphics_compiler/pubspec.yaml

```diff
@@ -32,7 +32,7 @@ dev_dependencies:
     sdk: flutter
   test: ^1.20.1
   # See https://github.com/flutter/flutter/issues/157626
-  vector_graphics: ">=1.1.11+1 <= 1.1.12"
+  vector_graphics: ^1.1.13
```

Comment:
Oops, we missed cleaning this one up in https://github.com/flutter/packages/pull/8018. Could you remove the comment on the previous line? It no longer applies once this is a regular `^` dependency.